packages/sdk/src/controllers/SubscriberController.ts

```diff
@@ -1,6 +1,6 @@
-import { ConfigType, Id } from '../types/CommonTypes';
-import { MeasurementUnit } from '../types/LayoutTypes';
-import { ToolType } from '../utils/enums';
+import {ConfigType, Id} from '../types/CommonTypes';
+import {MeasurementUnit} from '../types/LayoutTypes';
+import {ToolType} from '../utils/enums';
```

Comment:
guys, is there something wrong with my prettier?

Reply:
Indeed looks like something is wrong here, @brapoprod any idea?

Reply:
I got it, the path Prettier package has changed since the project has been restructured